src/services/ReportGenerationService.ts

```diff
@@ -62,11 +62,14 @@ class ReportGenerationService {
                 return template.workbook.xlsx.writeBuffer()
                 .then((buffer: Excel.Buffer) => {
                     return {
-                        fileName: `ATFReport_${moment(activity.startTime).tz(TimeZone.LONDON).format("DD-MM-YYYY")}_${moment(activity.startTime).tz(TimeZone.LONDON).format("HHmm")}_${activity.testStationPNumber}_${activity.testerName}.xlsx`,
-                        fileBuffer: buffer
+                        /* tslint:disable */fileName: `ATFReport_${moment(activity.startTime).tz(TimeZone.LONDON).format("DD-MM-YYYY")}_${moment(activity.startTime).tz(TimeZone.LONDON).format("HHmm")}_${activity.testStationPNumber}_${activity.testerName}.xlsx`,
+                        fileBuffer: buffer,
+                        testResults
                     };
                 });
             });
+        }).catch((error: any) => {
+            console.log(error);
```

Comment:
Same question about error throwing as above

Reply:
Should still throw error?

Reply:
Fixed